LabelPosition in IgxCheckBox is now only exported as a type, not a constant

## Description  
`LabelPosition` is now exported only as a type [`checkbox/public_api.ts`](https://github.com/IgniteUI/igniteui-angular/blob/1e50f39956fc54e14d4875e490bfe6cdadfdb5bc/projects/igniteui-angular/src/lib/checkbox/public_api.ts#L1C1-L2C38):

```typescript
export { type LabelPosition, IChangeCheckboxEventArgs } from "./checkbox-base.directive";
export * from "./checkbox.component";
```

Previously, it was also exported as a constant, which is a breaking change. In apps where it was used as a constant, this will now cause an error: `[ERROR] TS1362: 'LabelPosition' cannot be used as a value because it was exported using 'export type'. [plugin angular-compiler]`:

![Image](https://github.com/user-attachments/assets/9618d169-b20b-4d28-8152-2a8614767d1b)

 * igniteui-angular version: 19.1.x
 * browser:  any

## Result  
After updating to 19.1.x, an error is thrown in apps where `LabelPosition` was used as a constant.

## Expected result  
The error should not be thrown.

--- a/projects/igniteui-angular/src/lib/checkbox/public_api.ts
+++ b/projects/igniteui-angular/src/lib/checkbox/public_api.ts
@@ -1,2 +1,2 @@
-export { type LabelPosition, IChangeCheckboxEventArgs } from "./checkbox-base.directive";
+export { LabelPosition, IChangeCheckboxEventArgs } from "./checkbox-base.directive";
 export * from "./checkbox.component";

